pkg/controller/network/publicipaddress/managed.go

```diff
@@ -122,7 +130,15 @@ func (e *external) Create(ctx context.Context, mg resource.Managed) (managed.Ext
 }
 
 func (e *external) Update(ctx context.Context, mg resource.Managed) (managed.ExternalUpdate, error) {
```

Comment:
Can you add a test for this function? I believe it'd be almost same as `Create` one.